sdk/storage/Azure.Storage.Files.Shares/src/Models/ShareFileRangeInfo.cs

```diff
@@ -45,12 +50,20 @@ internal ShareFileRangeInfo(ShareFileRangeInfoInternal rangesInfoInternal)
             _shareFileRangeInfoInternal = rangesInfoInternal;
 
             // convert from internal Range type to HttpRange
-            var ranges = new List<HttpRange>();
-            foreach (Range range in rangesInfoInternal.Ranges)
+            List<HttpRange> ranges = new List<HttpRange>();
+            foreach (FileRange range in rangesInfoInternal.Body.Ranges)
             {
                 ranges.Add(new HttpRange(range.Start, range.End - range.Start + 1));
             }
             Ranges = ranges;
+
+            List<HttpRange> clearRanges = new List<HttpRange>();
```

Comment:
just want to double check this change will not affect GetRangeList api right?

Reply:
It will now appear in the output for GetRangeList, same as what happens in Page Blob.